.github/workflows/ci.yml

```
@@ -43,22 +45,22 @@ jobs:
     strategy:
       matrix:
         python:
-          - "3.10"
-          - "3.11"
+          - "3.12"
+          - "3.13"
         ansible:
-          - 6
-          - 7
-          - 8
+          - 10
+          - 11
         distro:
           - ubuntu2004
           - ubuntu2204
+          - ubuntu2404
     steps:
       - name: Checkout repository
         uses: actions/checkout@v4
         with:
           path: metabrainz.docker
 
-      - name: Set up Python ${{ matrix.python }}
+      - name: Set up Python 3
```

Maybe we should keep Python actual version in the name (as it was before)

```suggestion
      - name: Set up Python ${{ matrix.python }}
```